roles/openshift_repos/tasks/main.yaml

```diff
@@ -1,13 +1,20 @@
 ---
+- set_fact:
+    deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
```

Comment:
Instead of defining both deployment_type and openshift_deployment_type, you can nest default filters:
```
deployment_type: "{{ openshift_deployment_type | default(deployment_type | default(None)) }}"
```